frontend/packages/console-shared/src/components/lists/DetailPropertyList.scss

```diff
@@ -0,0 +1,12 @@
+ul.pf-c-list:not(.pf-m-inline).co-detail-property-list {
+  list-style-type: none;
+  list-style: none;
+  padding-left: 0;
+  li + li {
```

Comment:
why do we need this?

Reply:
This is to eliminate margin between `li`s defined in `.pf-c-list li+li ` rule.